LICENSE

```diff
@@ -1,5 +1,6 @@
 MIT License
 
+Copyright (c) 2023 BSV Blockchain Association ("BA")
```

Comment:
Maybe 2023-2024? Ask @dorzepowski

Reply:
the 2023 is enough according to the description of Copyright notice, it is a year of first publication under those rights.